README.md

```diff
@@ -1 +1,54 @@
-# config-monorepo
\ No newline at end of file
+# Eslint
+
+Provide reusable typescript and react eslint configs
+
+1. Install the peerDependencies of eslint configs:
+
+```sh
+npx install-peerdeps --dev @arbitrum/eslint-config-typescript/typescript
```

Comment:
Is it possible that all `npx / npm `commands be replaced with yarn since we use that globally?

Reply:
Got it! Thanks for this info!